nab/detectors/gaussian/windowedGaussian_detector.py

```diff
@@ -18,9 +18,33 @@
 # http://numenta.org/licenses/
 # ----------------------------------------------------------------------
 
-import numpy as np
+import math
+import numpy
+
 from nab.detectors.base import AnomalyDetector
-from nupic.algorithms import anomaly_likelihood
+
+
+
+def normalProbability(x, distributionParams):
+  """
+  Given the normal distribution specified by the mean and standard deviation in
+  distributionParams, return the probability of getting samples > x. This is the
+  Q-function: the tail probability of the normal distribution.
+
+  :param distributionParams: dict with values for keys "mean" and "stdev"
+  """
+  if "mean" not in distributionParams or "stdev" not in distributionParams:
+    raise RuntimeError("Insufficient parameters to specify the distribution.")
+
+  if x < distributionParams["mean"]:
+    # Gaussian is symmetrical around mean, so flip to get the tail probability
+    xp = 2*distributionParams["mean"] - x
+    return 1.0 - normalProbability(xp, distributionParams)
+
+  # Calculate the Q function with the complementary error function, explained
+  # here: http://www.gaussianwaves.com/2012/07/q-function-and-error-functions
+  z = (x - distributionParams["mean"]) / distributionParams["stdev"]
+  return 0.5 * math.erfc(z/math.sqrt(2))
```

Comment:
That's nice - didn't know erfc was in math.  Hopefully it is not too slow compared to a lookup table?